e2e/_suites/ingest-manager/fleet.go

```diff
@@ -270,7 +285,7 @@ func (fts *FleetTestSuite) theAgentIsUnenrolled() error {
 func (fts *FleetTestSuite) theAgentIsNotListedAsOnlineInFleet() error {
 	log.Debug("Checking if the agent is not listed as online in Fleet")
 
-	maxTimeout := 10 * time.Second
+	maxTimeout := time.Minute
```

Comment:
We increased the timeout as, locally, it takes around >30 secs to reach the "Online" status